DER/DerivativesContracts/Options.rdf

```diff
@@ -620,45 +512,46 @@
 	<owl:Class rdf:about="&fibo-fbc-fi-fi;Option">
```

Comment:
why is Option not definedBy the Options ontology??

Reply:
@rivettp The top level of our hierarchy for instruments including several derivatives, is defined in FBC and has been for years.  The definitions in FBC were the starting point from the CFI broad asset classes that include option.  This is not the only case, in other words.